### ISSUE TITLE
Deploy to public web space

### ISSUE THREAD

**Issue by limikael** [tunapanda/swagmaps#5]
Find us a place where we can host demos of our swagmaps, and create a mechanism for deploying to that place. Also, as a second step, integrate with travis-ci or similar so the demo is auto deployed there on commit.

**Comment by usernamenumber** [comment]
One option is I already have a Dreamhost VPS. If we don't need to handle tons of traffic I'd be willing to give y'all accounts, and then Tunapanda could save some hosting fees.

**Comment by usernamenumber** [comment]
...though maybe kili.io would have better performance for people in Africa? Or would our target audience for this stuff mostly be outside Africa?

**Comment by jaylarson85** [comment]
My guess is that Kili is better in East Africa, not sure about Africa-wide (I think it depends partially on the undersea cables http://www.submarinecablemap.com/). I think a larger percent of our target audience that would care as much about latency will be outside Africa.

We can also use our Bluehost account.

**Comment by adamn** [comment]
Up to you guys but Kili will be best for Kenyan users for sure.  Maybe develop on Dreamhost and then host on Kili later when it's ready for actual users (presuming that most are in East Africa)?  Kili ends up being about $15/month for a small instance and we can give you some credits if it helps.  Maybe it's best to just do both and see what works best ... and maybe even keep both up so you can play around with having a master system and an offline system (which could be simulated by routing traffic to one or the other servers into nowhere).

**Comment by usernamenumber** [comment]
Yeah, given that all the site code will be in github and there's no (or at
most a shared) back-end, running it in two places (maybe demo-east and
demo-west.tunapanda.org or something like that) seems do-able to me.
